app/src/responsive.sass

```diff
@@ -1 +1,6 @@
-$breakpoint-tablet: 500px
+//phone-size:  320px and 480px
+$breakpoint-phone: 480px
+//tablet-size: 481px and 768px
+$breakpoint-tablet: 768px
+//laptop-size: 769px and 1024px
+$breakpoint-laptop: 1024px
```

Comment:
I don't like this limit, because it's too close to the max-width of 960px of the title bar.